__tests__/app.test.js

```diff
@@ -73,3 +74,34 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+    test('200: responds with array of all article objects', () => {
+        return request(app).get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                expect(response.body.length).toBe(13)
+                expect(Object.keys(response.body[0])).toEqual([
+                    "article_id",
+                    "title",
+                    "topic",
+                    "author",
+                    "created_at",
+                    "votes",
+                    "article_img_url",
+                    "comment_count"
+                ]);
+                
+        })
+    })
+})
```

Comment:
Your test should also check the sort order, consider using jest-sorted to check this! 

I would say just checking the first result has the keys you expect doesn't really prove that all of the articles have the keys you expect also. We also aren't checking what the values are or what data type they should be. 

Consider trying to use a forEach here to look at each article and then using something like hasOwnProperty 😄 

Something along the lines of 
articles.forEach((article) => {
expect(article).hasOwnProperty("article_id", expect.any(Number))

etc. etc. for each property you expect to find 😄